### PR TITLE
AU Core Patient changes to add Must Support to Individual Gender Identity and Individual Pronouns 

### PR DESCRIPTION
**Pull request by dbojicic**
Addresses https://jira.hl7.org/browse/FHIR-43835 and https://jira.hl7.org/browse/FHIR-43836

1. Change AU Core Patient to place Mustr Support element on Individual Gender Identity and Individual Pronouns extensions
3. Brought across from AU Base Patient implementation guidance on Individual Gender Identity
4. Updated Gender Identity and related concepts
5. Added a note on the difference in gender identity with US Core in AU Core Patient intro.md and Comparison with other specs
6. Updated the diff on the home page
7. Another PR is coming to add examples for the two extensions

### CODE REVIEW

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The following are supported usage scenarios for this profile:
 A resource conforming to this profile is conformant to:
 - [IPA-Patient](http://hl7.org/fhir/uv/ipa/StructureDefinition-ipa-patient.html)
 - [Patient (IPS)](http://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Patient-uv-ips.html)
-- [US Core Patient](http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient)
+- [US Core Patient](http://hl7.org/fhir/us/core/StructureDefinition/us-core-patient) noting that US Core Patient includes US Gore Gender Identity extension that is not required to be supported
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-core-patient-intro.md:17]: Change US Core bit to 
if gender identity is not exchanged. US Core includes the US Gore Gender Identity extension; AU Core requires support of the international Gender Identity extension"

File: input/pagecontent/comparison.md [hunk 2]
```diff
@@ -909,7 +909,7 @@ Observation.subject.reference, Observation.status is                    <code>fi
             </td>
             <td width="111" valign="top">
                 <p>
-                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/>
+                    <img src="https://hl7.org/fhir/R4/assets/images/tick.png"/> noting that US Core Patient includes US Gore Gender Identity extension that is not required to be supported
```
Comment (dt-r) [inline input/pagecontent/comparison.md:912]: Update this text with the above from the intro.md

**Review by dt-r — CHANGES_REQUESTED** [review]
See comments.

File: input/intro-notes/StructureDefinition-au-core-patient-intro.md (hunk 1 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-core-patient-intro.md]: Done

File: input/pagecontent/comparison.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/comparison.md]: Done

**Comment by dbojicic** [comment]
@dt-r , applied changes, please review again.

**Review by dt-r — APPROVED** [review]

**Pull request merged**